test/math/FloatMatrixTest.ooc

```diff
@@ -145,7 +152,7 @@ FloatMatrixTest: class extends Fixture {
 		// 2 5
 		for (x in 0 .. matrix dimensions width) {
 			for (y in 0 .. matrix dimensions height) {
-				expect(matrix[x, y], is equal to(values[x * matrix dimensions height + y]))
+				expect(matrix[x, y], is equal to(values[x * matrix dimensions height + y]) within(this tolerance))
```

Comment:
Gah, now the original thing works on both platforms. I think the original problem was `this tolerance` was accidentally a `Double` which is problematic sometimes, but not always. Hang on.